apache-rat-core/src/main/java/org/apache/rat/Defaults.java

```diff
@@ -57,6 +62,10 @@ public class Defaults {
     public static final String UNAPPROVED_LICENSES_STYLESHEET = "org/apache/rat/unapproved-licenses.xsl";
 
     private final LicenseSetFactory setFactory;
+
+    private final FilenameFilter filesToIgnore = WildcardFileFilter.builder().setWildcards("*.json").setIoCase(IOCase.INSENSITIVE).get();
+
+    private final IOFileFilter directoriesToIgnore = NameBasedHiddenFileFilter.HIDDEN;
```

Comment:
Would it make sense to add these 2 special cases to the documentation?
Didn't you (or JB) add a configuration option to scan for hidden files?

Reply:
Should we create a new class for these 2 members? ScanDefaults that contains files and directories to ignore?

ScanDefault {
List<String> filesToIgnore; // wildcards
List<IOFileFilter> directoriesToIgnore; // not sure if IOFileFilter is the correct superclass
}

WDYT?

This would allow to have a static version of this configuration set in Defaults.java such as

public static ScanDefault RAT_DEFAULT_SCAN = new ScanDefault(List.of(*.json),List.of(NamebasedHiffenFilterFilter.HIDDEN);